test/webhooks/validation_test.go

```diff
@@ -71,6 +71,9 @@ func checkValidation(emqx v1beta3.Emqx) {
 	emqx.SetImage("emqx/emqx:latest")
 	Expect(k8sClient.Create(context.Background(), emqx)).Should(Succeed())
 
+	emqx.SetImage("127.0.0.1:8443/emqx/emqx:4.3.3")
+	Expect(k8sClient.Create(context.Background(), emqx)).ShouldNot(Succeed())
```

Comment:
In addition failed case, maybe add a successfully case is better, like "127.0.0.1:8443/emqx/emqx:4.4.3"